src/olympia/zadmin/tasks.py

```diff
@@ -587,14 +587,30 @@ def fetch_langpack(url, xpi, **kw):
             log.info('[@None] Created new "{0}" language pack, version {1}'
                      .format(xpi, data['version']))
 
+        # Set the category
+        for app in version.compatible_apps:
+            static_category = (
+                CATEGORIES[app.id][amo.ADDON_LPAPP]['general'])
```

Comment:
Android doesn't have any [langpacks](https://addons.mozilla.org/en-US/android/language-tools/).  For now we could just ignore the categories in the unlikely scenario one appears, but the point of the patch is to create complete add-ons that can be made public - when that work is finished the add-on can't be public so things are going to stop working, to some extent at least.